### PR TITLE
New patch release SLEPc 3.19.1

### PR DESCRIPTION
**Pull request by joseeroman**

### CODE REVIEW

**Review by balay — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-slepc4py/package.py
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-slepc4py/package.py:1]: Aren't we missing a numpy dependency? Not sure when it got added.

Comment (joseeroman) [reply var/spack/repos/builtin/packages/py-slepc4py/package.py:1]: `py-slepc4py` depends on `py-petsc4py` which in turn depends on `py-numpy`. If you prefer, I can add an explicit dependency also in `py-slepc4py`.

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-slepc4py/package.py:1]: Explicit is always better

Comment (joseeroman) [reply var/spack/repos/builtin/packages/py-slepc4py/package.py:1]: Done

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**